tests/draft2019-09/if-then-else.json

```diff
@@ -224,5 +224,35 @@
                 "valid": true
             }
         ]
+    },
+    {
+        "description": "if appears at the end when serialized (keyword processing sequence)",
+        "schema": {
+            "then": { "const": "yes" },
+            "else": { "const": "other" },
+            "if": { "maxLength": 4 }
+        },
+        "tests": [
+            {
+                "description": "yes redirects to then and passes",
+                "data": "yes",
+                "valid": true
+            },
+            {
+                "description": "other redirects to else and passes",
+                "data": "other",
+                "valid": true
+            },
+            {
+                "description": "no redirects to then and fails",
+                "data": "no",
+                "valid": false
```

Comment:
For this case, a validator processing the keywords *in serialization sequence* would process as follows.

`then` - no annotations from `if` -> skip
`else` - no annotations from `if` -> skip
`if` - produces no failure condition, just annotation for `then` and `else` to read -> valid

But this is invalid, so the test should fail for such a validator.